walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,99 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Set;
+import java.util.Collections;
+import java.util.Comparator;
 
 public final class FindMeetingQuery {
+
+  static final int startOfDay = TimeRange.START_OF_DAY;
+  static final int endOfDay = TimeRange.END_OF_DAY;
```

Comment:
Since these are constants, you should use CAPITAL_CAMEL_CASE

Reply:
changed